src/main/java/org/benetech/servicenet/service/OrganizationMatchService.java

```diff
@@ -58,4 +66,10 @@
     void dismissOrganizationMatch(UUID id, DismissMatchDTO dismissMatchDTO);
 
     void revertDismissOrganizationMatch(UUID id);
+
+    void hideOrganizationMatch(UUID id);
+
+    void hideOrganizationMatch(List<UUID> ids);
```

Comment:
I think this should be hideOrganizationMatches

Reply:
fixed